applications/GeoMechanicsApplication/tests/test_absorbing_boundary.py

```diff
@@ -106,11 +106,14 @@ def run_and_assert_1d_column(self, file_path, node_nbrs, direction):
             expected_ini_time = dist/self.vp
 
             # find index of expected time of wave arrival in time list
-            ini_time_idx = test_helper.find_closest_index_greater_than_value(res["time"], expected_ini_time)
+            res_keys = list(res.keys())
+            ini_time_idx = test_helper.find_closest_index_greater_than_value(res_keys, expected_ini_time)
```

Comment:
@mcgicjn2 These changes seem to give trouble in the pipeline for the slow tests (see [run logs](https://dpcbuild.deltares.nl/buildConfiguration/GEOFEA_KratosGeo_RunTests/2827080?expandBuildProblemsSection=true&hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&showLog=2827080_769_665&logView=flowAware))
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/f8e0365c-e8f4-4e35-a08f-5fd69363ebfa)

I'm not too sure what the intentions were of making this change, maybe you can have a look at it?

Reply:
I had made changes which appear to have been reverted which meant that the dictionary output was more structured, i.e. first index is time_step, 2nd is node and then 3rd is the value of the variable, this was needed in the transient time stepping. Here I was just updating all tests which used this facility to operate correctly. It was failing due to a revert that must have occured during the review process. I have now re-added the functionality.

Reply:
Okay, could be it happened when bringing the branch up to date with master, or I changed something I shouldn't have during the refactoring. In any case, thanks for double checking the PR and fixing this issue!